docs/pages/components/breadcrumb.md

```diff
@@ -41,19 +41,19 @@ there should be popover with missing options.
                     <span class="fd-breadcrumb__dropdown-icon"></span>
                 </a>
             </div>
-            <div class="fd-popover__body fd-popover__body--no-arrow" aria-hidden="true" id="breadcrumb1">
-                <ul class="fd-list fd-list--compact">
-                    <li class="fd-list__item">
-                      <a class="fd-list__title" href="#">Link Text</a>
+            <div class="fd-popover__body fd-popover__body--no-arrow fd-popover__body--no-overflow fd-popover__body--dropdown" aria-hidden="true" id="breadcrumb1">
+                <ul class="fd-list fd-list--compact fd-list--dropdown" role="listbox">
+                    <li class="fd-list__item fd-list__item--anchor" tabindex="-1" role="option">
+                      <a class="fd-list__item fd-list__title fd-list__title--anchor" tabindex="0" href=""><span class="">Link Text</span></a>
```

Comment:
The solution with `anchor` would work only in the case where a li item has ONLY text. But list item can have icons (before/after), it could be a whole container like in the case of Byline. I am currently working on List component so I suggest you park this issue for now and wait for my PR to be merged. If the issue still exists after that you can continue working on it

Reply:
Hmm alright, i think the li would have to hold a div then so ill wait for your PR.